src/game/engine/player.py

```diff
@@ -0,0 +1,61 @@
+from typing import Tuple, Final
+
+import pygame  # type: ignore
+
+
+class Player:
+    """
+    Player class allows for movement of player character and rendering
+
+    Parameters
+    ----------
+
+    position: Tuple[int, int, int, int]
+         starting (x, y, width, height) of player
+
+    image_path: str
+         path to image to use as player image
+    """
+
+    # position: x, y, width, height
+    position: Tuple[int, int, int, int]
```

Comment:
Does it makes sense to store width/height with position?

In the end we probably want the following vectors/tuples
- position: (X,Y)
- velocity: (xVel,yVel) (For jump moving on the screen/jump physics)
- acceleration: (xAccel, yAccel) (small xAccel for scaling difficulty, yAccel for jump physics)
- dimensions: (Width,Height)

Reply:
Pygames uses it, so I did too! See `player.py:draw`